ui/src/fixturemanager.cpp

```diff
@@ -1515,6 +1644,24 @@ void FixtureManager::slotRemap()
     updateView();
 }
 
+void FixtureManager::slotHighlightFixtures()
+{
+    if(m_highlightFixturesEnabled)
+    {
+        m_highlightFixturesAction->setIcon(QIcon(":/fixture.png"));
+        m_highlightFixturesAction->setIconText(tr("Enable highlight fixtures"));
+        m_highlightFixturesAction->setToolTip(tr("Enable highlight fixtures"));
+    } else
```

Comment:
Code style: else on newline. Parenthesis always alone on the row

Reply:
Fixed and note for future reference 👌